third_party/libevent/METADATA

```diff
@@ -6,13 +6,13 @@ description:
 third_party {
   url {
     type: GIT
-    value: "https://github.com/libevent/libevent.git"
+    value: "https://chromium.googlesource.com/chromium/src"
```

Comment:
I'd suggest move the `https://github.com/libevent/libevent.git` into a comment, not fully deleting it.

```
# Upstream: https://github.com/libevent/libevent.git
```

Note METADATA schema actually allows multiple urls to be specified with different types, too, but we don't have to invent how we use that here - that's a longer design conversation.

Reply:
A comment seems fine for now at least.